tests/data_files/.gitattributes

```diff
@@ -0,0 +1 @@
+*.bin -text
```

Comment:
I don't think this should be needed? It seems like a local git configuration issue rather than anything PKCS7 specific?